Mathlib/RingTheory/OreLocalization/Ring.lean

```diff
@@ -0,0 +1,267 @@
+/-
+Copyright (c) 2022 Jakob von Raumer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob von Raumer, Kevin Klinge, Andrew Yang
+-/
+
+import Mathlib.Algebra.GroupWithZero.NonZeroDivisors
+import Mathlib.Algebra.Module.Defs
+import Mathlib.Algebra.Field.Defs
+import Mathlib.RingTheory.OreLocalization.Basic
+
+#align_import ring_theory.ore_localization.basic from "leanprover-community/mathlib"@"861a26926586cd46ff80264d121cdb6fa0e35cc1"
+
+/-!
+
+# Module and Ring instances of Ore Localizations
```

Comment:
You could perhaps add here a sentence saying that alghough here everything is multiplicative, there exists an additive version in `bla/bla/file.lean`.